src/views/index_view.js

```diff
@@ -113,20 +113,20 @@ export default BaseView.extend({
   },
 
   // Adjust the height of the iframe to match the height of the body.
-  adjustHeight () {
+  adjustHeight() {
     webChannel.sendAdjustHeight(this.el.offsetHeight);
   },
 
   // Clear results of each of our adapters.
-  resetAdapters () {
-    let adapters = [
+  resetAdapters() {
+    const adapters = [
       activityResultsAdapter.results,
       recommendedAdapter.results,
       searchSuggestionsAdapter.localSuggestions,
       searchSuggestionsAdapter.remoteSuggestions,
       searchSuggestionsAdapter.combinedSuggestions
     ];
-    for (let adapter in adapters) {
+    for (const adapter in adapters) { // eslint-disable-line guard-for-in
```

Comment:
This is gross and makes me feel gross. If I don't disable the `guard-for-in` ESLint rule here, I get the following error:

```
/Users/pdehaan/dev/github/universal-search-content/src/views/index_view.js
  129:5  error  The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype  guard-for-in

✖ 1 problem (1 error, 0 warnings)
```

Not sure if we want to:
a) Ignore the `guard-for-in` rule above.
b) Wrap body if an `if` to filter unwanted things.
c) Convert to a `forEach()` loop.